exe/scarpe

```diff
@@ -20,9 +18,7 @@ end
 
 require "scarpe"
 
-if use_shoes
-    Shoes = Scarpe
-end
+Shoes = Scarpe unless defined?(Shoes)
```

Comment:
I'd probably move this into the framework, like in lib/scarpe.rb or a similar spot.

Reply:
Oh hey, I just realised some places we're doing things not-perfectly in the framework. I retract the above suggestion - we need to fix the framework side before we move this elsewhere. So this is fine, don't change it in this PR.

Reply:
so the only thing that shoould be updated in this PR as of now is `README.md `right?

Reply:
and uncommenting ruby_version>3,2 on in `scarpe.rb`

Reply:
Yes, just those things.